src/pages/namespace/Explorer/Workflow/Revisions/Overview/TrafficShaping/RevisionSelector.tsx

```diff
@@ -7,16 +7,16 @@ import {
   CommandItem,
   CommandList,
   CommandStaticItem,
-} from "../../../../../../design/Command";
+} from "../../../../../../../design/Command";
```

Comment:
should we use a `~/` path here instead? (also applies to the other imports in this file)